src/pipeline/src/manager/table.rs

```diff
@@ -49,6 +49,8 @@ use crate::error::{
 use crate::etl::transform::GreptimeTransformer;
 use crate::etl::{parse, Content, Pipeline};
 
+pub type PipelineVersion = Option<TimestampNanosecond>;
```

Comment:
nitpick: Document that `None` means latest.

Reply:
Good idea!